### DISCUSSION TITLE
Maybe we need the additional terms for pressure stability in SI.

### DISCUSSION THREAD

**Discussion by Goldeninscriptions** [APSIS-ANALYSIS/PERIGEE#188] [Ideas]
<img width="457" alt="Screenshot 2024-11-08 at 17 34 05" src="https://github.com/user-attachments/assets/333941ec-7e0b-4c86-bae9-f830c44106fa">
<img width="454" alt="Screenshot 2024-11-08 at 17 26 09" src="https://github.com/user-attachments/assets/ff0836d5-7344-4308-9cb1-ab20d70c11be">

See Pages 207~208 of CF-2017-Compressible. 万一旧格式根本就只管了速度没管压强呢？

**Comment by ju-liu** [comment]
压强作为应力的一部分被黏合了...不应该有问题。

压强还是震荡？

交界面上有压强不连续吗？

**Comment by Goldeninscriptions** [comment]
压强的震荡是在转起来之后的整个域的全局震荡，交界面上是连续的。

孙师兄又用最新的程序，重新算了固定大圆筒套旋转小圆筒、入流稳定出口0压强的例子。
发现一开始压强还正常，转久了之后全局的压强也乱了。

**Reply by Goldeninscriptions** [reply]
之前两边traction转起来又没见过震荡，可能和出口0压强的边界条件也有关系
